app/players/elves/druid.py

```diff
@@ -0,0 +1,18 @@
+from app.players.elves.elf import Elf
+
+
+class Druid(Elf):
+    def __init__(self,
+                 nickname: str,
+                 musical_instrument: str,
+                 favourite_spell: str) -> None:
+        super().__init__(nickname, musical_instrument)
+        self._favourite_spell = favourite_spell
+
+    def player_info(self) -> None:
+        return (f"Druid {self.nickname}. "
+                f"{self.nickname} has a favourite pell: "
+                f"{self._favourite_spell}")

```

Comment:
There is a typo in the `player_info` method. The word 'pell' should be corrected to 'spell'. The method should return: `"Druid {nickname}. {nickname} has a favourite spell: {favourite_spell}"` as specified in the task description.